nums/core/array/application.py

```diff
@@ -751,17 +751,18 @@ def get(self, *arrs):
                     r.append(item)
             return r
 
-    def allclose(self, a: BlockArray, b: BlockArray, rtol=1.e-5, atol=1.e-8):
+    def array_eq(self, a: BlockArray, b: BlockArray, func_name, *args):
```

Comment:
Can you update the signature for this to be `self, func_name: str, a: BlockArray, b: BlockArray, *args, **kwargs`? It's a bit more consistent with existing methods on this class.

Reply:
Will do

Reply:
Do we need `**kwargs` in the function signature? It doesn't seem like it would be used here.

Reply:
Yes, if it's not going to be used leave it out.